torch/_torch_docs.py

```diff
@@ -6069,6 +6069,53 @@ def merge_dicts(*dicts):
     tensor([  2.,   4.,   8.,  16.])
 """.format(**common_args))
 
+add_docstr(torch.float_power,
+           r"""
+float_power(input, exponent, *, out=None) -> Tensor
+
+Raises :attr:`input` to power of :attr:`exponent`, elementwise, in double precision. 
+If neither input is complex returns a ``torch.float64`` tensor, 
+and if one or more inputs is complex returns a ``torch.complex128`` tensor.
+
+:attr:`exponent` can be either a single ``float`` number or a `Tensor`
```

Comment:
Is this true? Does pow not support complex exponents or broadcasting of the exponent?

Reply:
Fixed, thanks for correction！